danger-kotlin/src/runnerMain/kotlin/systems.danger/cmd/dangerfile/DangerFile.kt

```diff
@@ -57,7 +57,7 @@ private fun readLine(file: CPointer<FILE>): String? {
     var ch = getc(file)
     var lineBuffer: Array<Char> = arrayOf()
 
-    while ((ch != '\n'.toInt()) && (ch != EOF)) {
+    while ((ch != '\n'.code) && (ch != EOF)) {
```

Comment:
@gianluz this should probably be updated in #231

Reply:
done :smile: Thanks